heroku/cli
# refactor(container-registry-v5): move container:rm to CLI & upgrade to oclif

### Description

This converts `container:rm` to oclif and updates its associated test.

### Testing

1) Create an app based on a Dockerfile with a web process
2) Run `./bin/run container:rm web -a APP_NAME` with this branch pulled down
3) Verify the command succeeds and the web process is removed from the app
3) Compare the results with prod cli command and make sure things look 👍
